test/integration/forms/form.js

```diff
@@ -516,6 +577,7 @@ context('Patient Action Form', function() {
 
         return fx;
       })
+      .routePatientField()
```

Comment:
Unrelated to this PR. But noticed a `500` error in this test because this was missing.